reo/tests/test_chp.py

```diff
@@ -61,3 +61,63 @@ def test_chp_sizing(self):
             print("Error message: {}".format(d['messages'].get('error')))
             raise
 
+    def test_cost_curve(self):
```

Comment:
@Bill-Becker As you mentioned in the commit summary this new CHP cost curve capability did not make any existing tests slower. That is to be expected since no existing tests use the new capability. It would be interesting to know if you used the cost curve in `test_chp_sizing` what impact it has on the solve time.  If it doesn't significantly impact that test's time then we could integrate your new test with the existing one so that there is little net impact on the time it takes for the total test suite to run. Just a suggestion and not necessary.

Reply:
@NLaws The testing I did was the ~120x scenario benchmark testing using the monolith (which we're going with for the baseline model until we improve decomposition), and there was an overall positive benefit to time/non-timeout/non-negative-NPV. I documented this in the benchmark testing doc on Box. Good idea to combine CHP tests as this will be the new default cost input for CHP.